draft-ietf-tls-tls13.md

```diff
@@ -2682,12 +2663,17 @@ The "extension_data" field of this extension contains a
                    psk_identity identities<2..2^16-1>;
 
                case server:
-                   psk_identity identity;
+                   uint16 selected_identity;
            }
        } PreSharedKeyExtension;
 
-identity
-: An opaque label for the pre-shared key.
+identities
```

Comment:
Would it make sense to use a different name for these.  It's a PSK identifier.  Using the word "identity" might still be a good idea to trigger the right privacy-related immune responses though.